Migrate Adding Inline Token test to Playwright

pooja-muchandikar opened the pull request:
<!-- Thanks for contributing to Gutenberg! Please follow the Gutenberg Contributing Guidelines:
https://github.com/WordPress/gutenberg/blob/trunk/CONTRIBUTING.md -->
## What?
<!-- In a few words, what is the PR actually doing? -->
Part of https://github.com/WordPress/gutenberg/issues/38851.
Migrate[ adding-inline-tokens.test.js ](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/specs/editor/various/adding-inline-tokens.test.js) to its Playwright version.
## Why?
<!-- Why is this PR necessary? What problem is it solving? Reference any existing previous issue(s) or PR(s), but please add a short summary here, too -->
Part of https://github.com/WordPress/gutenberg/issues/38851.
## How?
<!-- How is your PR addressing the issue at hand? What are the implementation details? -->
See[ MIGRATION.md](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/MIGRATION.md) for migration steps.
## Testing Instructions
<!-- Please include step by step instructions on how to test this PR. -->
<!-- 1. Open a Post or Page. -->
<!-- 2. Insert a Heading Block. -->
<!-- 3. etc. -->
Run` npm run test:e2e:playwright test/e2e/specs/editor/various/adding-inline-tokens.spec.js`

pooja-muchandikar committed:
Migrate Adding Inline Token test to Playwright

pooja-muchandikar committed:
Add space

pooja-muchandikar commented:
Hi @kevin940726,

I hope you are going great!..

Could you please help me in reviewing this PR?

Thanks!!

kevin940726 commented on test/e2e/specs/editor/various/adding-inline-tokens.spec.js:
```diff
@@ -47,26 +40,21 @@ describe( 'adding inline tokens', () => {
 			`<!-- wp:paragraph -->\\s*<p>a <img class="wp-image-\\d+" style="width:\\s*10px;?" src="[^"]+\\/${ filename }\\.png" alt=""\\/?><\\/p>\\s*<!-- \\/wp:paragraph -->`
 		);
-		expect( await getEditedPostContent() ).toMatch( regex );
+		expect( await editor.getEditedPostContent() ).toMatch( regex );
```
Let's instead do something like:

```js
const contentRegex = new RegExp(
	`a <img class="wp-image-\\d+" style="width:\\s*10px;?" src="[^"]+\\/${ filename }\\.png" alt=""\\/?>`
);
await expect.poll( editor.getBlocks ).toMatchObject( [ {
	name: 'core/paragraph',
	attributes: { content: expect.stringMatching( contentRegex ) }
} ] );
```

kevin940726 commented on test/e2e/specs/editor/various/adding-inline-tokens.spec.js:
```diff
@@ -76,6 +64,6 @@ describe( 'adding inline tokens', () => {
 			`<!-- wp:paragraph -->\\s*<p>a <img class="wp-image-\\d+" style="width:\\s*20px;?" src="[^"]+\\/${ filename }\\.png" alt=""\\/?><\\/p>\\s*<!-- \\/wp:paragraph -->`
 		);
-		expect( await getEditedPostContent() ).toMatch( regex2 );
+		expect( await editor.getEditedPostContent() ).toMatch( regex2 );
```
Same here with a similar change.

kevin940726 reviewed (commented):
Thank you! Sorry for the late review! This is looking great though! Just a small suggestion 🙇‍♂️.

pooja-muchandikar committed:
Address Feedbacks

pooja-muchandikar commented:
Hi @kevin940726, 

I hope you are doing great!

I have addressed all the feedbacks and pushed the code, but seems PR has some conflicts however, I am not able to see these conflicts in my local. Could you please help in resolve it from PR directly or should I create new PR with the suggested changes?

pooja-muchandikar commented:
Hi @kevin940726,
I am closing this PR as the conflicts are not visible in my local because of which unable to fix it.. 

So I have created new PR with all the changes/feedbacks being addressed - https://github.com/WordPress/gutenberg/pull/52020

Please review and let me know if there are more suggestions!

Thanks!
